Sources/Operators.swift

```diff
@@ -16,7 +16,7 @@ import Foundation
 
 // MARK: - <| Operator
 
-infix operator <| { associativity left precedence 150 }
+infix operator <|
```

Comment:
I'm assuming we probably want this `infix operator |> : DefaultPrecedence` precedence level. Which apparently is the same as not adding one.